ajax/libs/interactive-data-display/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "interactive-data-display",
+  "filename": "idd.min.js",
+  "description": "A JavaScript visualization library for dynamic data",
+  "homepage": "https://github.com/predictionmachines/InteractiveDataDisplay/wiki",
```

Comment:
@ElenaPochernina I think it's better to use https://predictionmachines.github.io/InteractiveDataDisplay/ as `homepage`.